### PR TITLE
Fighting bots

### PR DESCRIPTION
**Pull request by AleksyBalazinski**
I left the hp counter in debug mode but it's nightmarishly ugly, so I can remove this if you like

### CODE REVIEW

File: Hyper/Controllers/Bots/Spawn/SphericalBotSpawnStrategy.cs [hunk 1]
```diff
@@ -37,7 +37,18 @@ public override void Spawn()
 
     public override void Despawn()
```
Comment (KarolDenst) [inline Hyper/Controllers/Bots/Spawn/SphericalBotSpawnStrategy.cs:38]: Do we maybe want to spawn bots if there are fewer than 7 in spherical?

File: Hyper/Controllers/Bots/Spawn/StandardBotSpawnStrategy.cs [hunk 2]
```diff
@@ -12,25 +12,24 @@ internal class StandardBotSpawnStrategy : AbstractBotSpawnStrategy
...
+    private readonly float _maxSpawnRadius;
 
     public StandardBotSpawnStrategy(Scene scene, Settings settings) : base(scene, settings)
     {
-        _maxBots = 10 * settings.RenderDistance * settings.RenderDistance * Chunk.Size / 32 * Chunk.Size / 32;
-        _minSpawnRadius = Chunk.Size * settings.RenderDistance / 3;
-        _maxSpawnRadius = Chunk.Size * settings.RenderDistance * 2 / 3;
+        _maxBots = 4 * settings.RenderDistance * settings.RenderDistance * Chunk.Size / 32 * Chunk.Size / 32;
+        _minSpawnRadius = Chunk.Size * settings.RenderDistance * 0.5f;
+        _maxSpawnRadius = Chunk.Size * settings.RenderDistance * 1.6f;
```
Comment (KarolDenst) [inline Hyper/Controllers/Bots/Spawn/StandardBotSpawnStrategy.cs:23]: doesn't this mean that bots can spawn outside of the generated terrain?

File: Hyper/Controllers/Bots/Spawn/SphericalBotSpawnStrategy.cs (hunk 1 above)
Comment (AleksyBalazinski) [reply Hyper/Controllers/Bots/Spawn/SphericalBotSpawnStrategy.cs:38]: Spawn method will generate a new bot once we kill one, so I don't think this is necessary

File: Hyper/Controllers/Bots/Spawn/StandardBotSpawnStrategy.cs (hunk 2 above)
Comment (AleksyBalazinski) [reply Hyper/Controllers/Bots/Spawn/StandardBotSpawnStrategy.cs:23]: It didn't break when I tested but that may have been blind luck. I'll change max to 1 (better safe than sorry) and min to 0.7 to give the player some chance to escape

**Review by KarolDenst — APPROVED** [review]

**Pull request merged**